api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -277,4 +277,6 @@
 	public static final String INCLUDE_RESULT_PARAM = "result";
 	
 	public static final String REVERSE_INCLUDE_SEARCH_HANDLER = "_revinclude.search.handler";
+	
+	public static final Integer CONDITION_OBSERVATION_CONCEPT_ID = 1284;
```

Comment:
It would be safer to use the UUID for the concept ("1284AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA") since we don't want to be relying on a concept having a given concept id (i.e., we don't want to rely on a dictionary having IDs in a certain order). Even better might be to just be to leverage the system for CIEL and the concept ID, via something like:

```java
TokenAndListParam problemList = new TokenAndListParam();
		problemList.addAnd(new TokenParam().setSystem("https://openconceptlab.org/orgs/CIEL/sources/CIEL").setValue("1284"));
		handleCodeableConcept(criteria, problemList, "c", "cm", "crt");
```

Then this "1284" string would have a place in the `FhirConstants` file.